pvlib/bifacial/infinite_sheds.py

```diff
@@ -0,0 +1,792 @@
+
+    ghi : numeric
+        Global horizontal irradiance. [W/m2]
+
+    dhi : numeric
+        Diffuse horizontal irradiance. [W/m2]
+
+    dni : numeric
+        Direct normal irradiance. [W/m2]
+
+    albedo : numeric
+        Surface albedo. [unitless]
+
+    iam_front : numeric, default 1.0
+        Incidence angle modifier, the fraction of direct irradiance incident
+        on the front surface that is not reflected away. [unitless]
+
+    iam_back : numeric, default 1.0
+        Incidence angle modifier, the fraction of direct irradiance incident
+        on the back surface that is not reflected away. [unitless]
+
+    bifaciality : numeric, default 0.8
+        Ratio of the efficiency of the module's rear surface to the efficiency
+        of the front surface. [unitless]
+
+    shade_factor : numeric, default -0.02
+        Fraction of back surface irradiance that is blocked by array mounting
+        structures. Negative value is a reduction in back irradiance.
+        [unitless]
+
+    transmission_factor : numeric, default 0.0
+        Fraction of irradiance on the back surface that does not reach the
+        module's cells due to module structures. A negative value is a
+        reduction in back irradiance. [unitless]
+
+    npoints : int, default 100
+        Number of points used to discretize distance along the ground.
+
+    Returns
+    -------
+    output : OrderedDict or DataFrame
+        Output is a DataFrame when input ghi is a Series. See Notes for
+        descriptions of content.
+
+    Notes
+    -----
+
+    ``output`` includes:
+
+    - ``poa_global`` : total irradiance reaching the module cells from both
+      front and back surfaces. [W/m^2]
+    - ``poa_front`` : total irradiance reaching the module cells from the front
+      surface. [W/m^2]
+    - ``poa_back`` : total irradiance reaching the module cells from the front
+      surface. [W/m^2]
+
+    References
+    ----------
+    .. [1] Mikofksi, M., Darawali, R., Hamer, M., Neubert, A., and Newmiller,
+       J. "Bifacial Performance Modeling in Large Arrays". 2019 IEEE 46th
+       Photovoltaic Specialists Conference (PVSC), 2019, pp. 1282-1287.
+       doi: 10.1109/PVSC40753.2019.8980572.
+
+    See also
+    --------
+    get_irradiance_poa
+    """
+    # backside is rotated and flipped relative to front
+    backside_tilt, backside_sysaz = _backside(surface_tilt, surface_azimuth)
+    # rows to consider in front and behind current row
+    max_rows = int(100 * gcr)
+    # front side POA irradiance
+    irrad_front = get_irradiance_poa(
+        surface_tilt, surface_azimuth, solar_zenith, solar_azimuth,
+        gcr, height, pitch, ghi, dhi, dni, albedo, iam_front, max_rows,
+        npoints)
+    # back side POA irradiance
+    irrad_back = get_irradiance_poa(
+        backside_tilt, backside_sysaz, solar_zenith, solar_azimuth,
+        gcr, height, pitch, ghi, dhi, dni, albedo, iam_back, max_rows,
```

Comment:
I think these `max_rows` are being passed to the `axis_azimuth` parameter.  Maybe we should switch to passing via keyword instead of position for these two calls to be safer.

Reply:
> @cwhanse following up from yesterday: on [b316d3b](https://github.com/pvlib/pvlib-python/commit/b316d3bd4ada591b676734d8986f16f3402bbcb2), modified to address the two issues above, I've not found a case where the value of `axis_azimuth` (`None` or any legitimate value) affects the results. So I think it is indeed a no-effect parameter. I vote to remove it for now and add it back in as needed for some potential future model extension.

Agreed. The case where it matters is when `axis_tilt` is not zero, but in that case, I don't know what `height` means. I think `infinite_sheds` is currently limited to an array with tracker axes parallel to the ground.